gen-ai/inference/llama3-8b-instruct-rayserve-inf2/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# docker buildx build --platform=linux/amd64 -t ray-serve-llama2:latest .
+FROM rayproject/ray:2.21.0-py310
+
+# Maintainer label
+LABEL maintainer="DoEKS"
+
+# Set environment variables to non-interactive (this prevents some prompts)
+ENV DEBIAN_FRONTEND=non-interactive
+
+# Switch to root to add Neuron repo and install necessary packages
+USER root
+
+# Set up the Neuron repository and install Neuron packages
+RUN . /etc/os-release && \
+    sudo echo "deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/neuron.list && \
+    sudo wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add - && \
+    sudo apt-get update -y && \
+    sudo apt-get install aws-neuronx-dkms aws-neuronx-collectives=2.* aws-neuronx-runtime-lib=2.* aws-neuronx-tools=2.* -y && \
```

Comment:
FYI - we should strip `aws-neuronx-dkms` from all of the docker images in this repo. that is the Neuron driver and its provided on the AMI/host